composer/trainer/trainer_hparams.py

```diff
@@ -195,6 +194,10 @@ def validate(self):
                 f"eval batch size ({self.eval_batch_size}) not divisible by the total number of processes ({world_size}) "
             )
 
+        # There should be at least one structure for validation - evaluators or ordinary validation dataset
+        if self.val_dataset is None and not self.evaluators:
```

Comment:
Hmm interesting point....but should this be an error? Or a warning (if there use cases for training without ever evaluating?)